packages/remote-viewport/src/remote-viewport.ts

```diff
@@ -0,0 +1,13 @@
+import { createViewport as parentCreateViewport } from '@itk-viewer/viewer/viewport.js';
+
+export const createViewport = ({
+  parent,
+  address,
+}: {
+  parent: HTMLElement;
```

Comment:
Return a DOM element that the owner is responsible for parenting somewhere.  Don't pass in parent.